### ISSUE TITLE
housekeeper does not list file present in bundle

### ISSUE THREAD

**Issue by seallard** [Clinical-Genomics/housekeeper#128]
## Description
`housekeeper add file` fails, stating that the file already exists. However, when the specific bundle is retrieved with `housekeeper get bundle` it is shown to be empty. When looking at the bundle directory, the file is present in a version - so it should be listed for the bundle. It cannot be retrieved with `housekeeper get file` either.

The command below was run in the `/home/proj/production/housekeeper-bundles/ADM1091A3/2018-06-05` directory:
```
for f in *; do housekeeper add file -t fastq -t H9GA6ADXX -b ADM1091A3 ./${f}; done
2023-06-13 09:47:51 [hasta.scilifelab.se](http://hasta.scilifelab.se/) housekeeper.cli.core[37109] INFO Use root path /home/proj/production/housekeeper-bundles
2023-06-13 09:47:51 [hasta.scilifelab.se](http://hasta.scilifelab.se/) housekeeper.cli.add[37109] INFO Running add file
2023-06-13 09:47:51 [hasta.scilifelab.se](http://hasta.scilifelab.se/) housekeeper.store.api.handlers.read[37109] INFO Fetching bundle with name: ADM1091A3
Traceback (most recent call last):
 File "/home/proj/production/bin/miniconda3/envs/P_main/bin/housekeeper", line 8, in <module>
  sys.exit(base())
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 722, in __call__
  return self.main(*args, **kwargs)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 697, in main
  rv = self.invoke(ctx)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
  return _process_result(sub_ctx.command.invoke(sub_ctx))
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
  return _process_result(sub_ctx.command.invoke(sub_ctx))
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 895, in invoke
  return ctx.invoke(self.callback, **ctx.params)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/core.py", line 535, in invoke
  return callback(*args, **kwargs)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
  return f(get_current_context(), *args, **kwargs)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/housekeeper/cli/add.py", line 124, in file_cmd
  link_to_relative_path(version=version, file_path=file_path, root_path=context.obj[ROOT])
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/housekeeper/include.py", line 63, in link_to_relative_path
  link_file(file_path=file_path, new_path=housekeeper_path, hardlink=True)
 File "/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/housekeeper/include.py", line 19, in link_file
  os.link(file_path.resolve(), new_path)
FileExistsError: [Errno 17] File exists: '/home/proj/production/housekeeper-bundles/ADM1091A3/2018-06-05/ADM1091A3_L001_R1_001.fastq.gz' -> '/home/proj/production/housekeeper-bundles/ADM1091A3/2018-06-05/ADM1091A3_L001_R1_001.fastq.gz'
```

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by seallard** [comment]
It is possible that the file was manually deleted from the bundle in the database.

**Comment by ChrOertlin** [comment]
duplicate of: https://github.com/Clinical-Genomics/housekeeper/issues/129
